lib/pyld/documentloader/requests.py

```diff
@@ -12,8 +12,7 @@
 import string
 import urllib.parse as urllib_parse
 
-from pyld.jsonld import (JsonLdError, parse_link_header, LINK_HEADER_REL)
-
+from pyld.jsonld import (JsonLdError, parse_link_header, prepend_base, LINK_HEADER_REL)
```

Comment:
```suggestion
from pyld.jsonld import JsonLdError, parse_link_header, prepend_base, LINK_HEADER_REL
```

I believe the braces are redundant.